library/core/src/cmp.rs

```diff
@@ -245,7 +245,7 @@ pub trait PartialEq<Rhs: ?Sized = Self> {
 }
 
 /// Derive macro generating an impl of the trait [`PartialEq`].
-/// The behavior of this macro is described in detail [here](PartialEq#derivable).
+/// The behavior of this macro is described in detail [here](https://doc.rust-lang.org/std/cmp/trait.PartialEq.html#derivable).
```

Comment:
We try to avoid explicit doc links and prefer intra-doc links. Doesn't the following fix the issue, too?


```suggestion
/// The behavior of this macro is described in detail [here](trait@PartialEq.html#derivable).
```

Reply:
I've just noticed that the link isn't broken for me. How can I reproduce this issue? All four of these work:

* https://doc.rust-lang.org/nightly/std/cmp/derive.PartialEq.html
* https://doc.rust-lang.org/nightly/core/cmp/derive.PartialEq.html
* https://doc.rust-lang.org/std/cmp/derive.PartialEq.html
* https://doc.rust-lang.org/core/cmp/derive.PartialEq.html

Reply:
The original link opens [https://doc.rust-lang.org/stable/core/cmp/PartialEq#derivable](https://doc.rust-lang.org/stable/core/cmp/PartialEq#derivable) for me.
I just clicked the link. I'm on:
cargo --version
cargo 1.73.0 (9c4383fb5 2023-08-26)
rustc --version
rustc 1.73.0 (cc66ad468 2023-10-03)

I know that my "solution" is not great but I tried your suggestion and it redirects me here:
[https://doc.rust-lang.org/stable/core/cmp/trait@PartialEq.html#derivable](https://doc.rust-lang.org/stable/core/cmp/trait@PartialEq.html#derivable)
Can you point me to the documentation of these links? I'll find a better way if I know the inner mechanics maybe...?